lux/core/series.py

```diff
@@ -211,7 +215,83 @@ def recommendation(self):
             self.name = " "
         ldf = LuxDataFrame(self)
 
-        if self._recommendation is not None and self._recommendation == {}:
```

Comment:
why is this if statement taken out?

Reply:
Recommendation is never set in series.py, and in frame.py the recommendation is always recomputed, just in case recommendation is called without calling repr_html first

Reply:
actually fixed now!